### PR TITLE
Http 203 podcast episode 9

### PR DESCRIPTION
**Pull request by jakearchibald**

### CODE REVIEW

**Comment by jakearchibald** [comment]
+@surma

**Comment by surma** [comment]
Lgtm

File: src/content/en/shows/http203/podcast/a-is-for-effort.md [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+
+In this episode:
+
+* Polymer summit happened, [watch the talks](https://summit.polymer-project.org/schedule).
+* Especially [Surma & Monica's supercharged](https://www.youtube.com/watch?v=tHJwRWrexqg).
+* How Jake [sweated through supercharged](https://www.youtube.com/watch?v=3Tr-scf7trE).
+* Service workers are being [implemented in Safari](https://bugs.webkit.org/show_bug.cgi?id=174541).
+* Jake built a [silly PWA](https://f1-start.glitch.me/), [endorsed by F1 world champions](https://www.youtube.com/watch?v=6fgGJ-M6X2s).
+* Surma built [Tinder for bananas](https://tinderforbananas.com/).
+* [CSS animation worklets and custom layouts](https://developers.google.com/web/updates/2016/05/houdini).
```
Comment (petele) [inline src/content/en/shows/http203/podcast/a-is-for-effort.md]: Do not hard code `developers.google.com` in links

File: src/content/en/shows/http203/podcast/a-is-for-effort.md [hunk 2]
```diff
@@ -0,0 +1,45 @@
+project_path: /web/_project.yaml
```
Comment (petele) [inline src/content/en/shows/http203/podcast/a-is-for-effort.md]: this should be in `src/content/en/shows/http203/podcast/series-3/` i assume?

File: src/content/en/shows/http203/podcast/a-is-for-effort.md [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+
+<img src="/web/shows/http203/podcast/images/surma-and-jake.jpg" class="attempt-right">
+
+In this episode:
+
+* Polymer summit happened, [watch the talks](https://summit.polymer-project.org/schedule).
+* Especially [Surma & Monica's supercharged](https://www.youtube.com/watch?v=tHJwRWrexqg).
+* How Jake [sweated through supercharged](https://www.youtube.com/watch?v=3Tr-scf7trE).
+* Service workers are being [implemented in Safari](https://bugs.webkit.org/show_bug.cgi?id=174541).
+* Jake built a [silly PWA](https://f1-start.glitch.me/), [endorsed by F1 world champions](https://www.youtube.com/watch?v=6fgGJ-M6X2s).
```
Comment (petele) [inline src/content/en/shows/http203/podcast/a-is-for-effort.md]: line exceeds 100 char

File: src/content/en/shows/http203/podcast/a-is-for-effort.md [hunk 4]
```diff
@@ -0,0 +1,45 @@
...
+* Surma built [Tinder for bananas](https://tinderforbananas.com/).
+* [CSS animation worklets and custom layouts](https://developers.google.com/web/updates/2016/05/houdini).
+* [CSS shadow parts](https://tabatkins.github.io/specs/css-shadow-parts/) for styling the shadow DOM.
+* What has Flash ever done for us?
+* Jake uploaded [an old Flash intro he built](https://cv-ppguyzgojs.now.sh/).
+* Do we cultivate impostor syndrome as if it's a good thing?
+* [Admitting what we don't know](https://twitter.com/jaffathecake/status/835963707368869890) is a good thing.
```
Comment (petele) [inline src/content/en/shows/http203/podcast/a-is-for-effort.md]: line exceeds 100 char

**Comment by jakearchibald** [comment]
Fixed

**Comment by jakearchibald** [comment]
Ta!

**Pull request merged**